CarRentals/CarRentals/Controllers/CarCRUD.cs

```diff
@@ -1,80 +1,60 @@
-﻿using CarRentals.Exceptions;
+﻿using CarRentals.Interfaces;
 using Models;
 using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using System.Text;
 using System.Text.Json;
 using System.Text.Json.Serialization;
+using System.Configuration;
 
 namespace CarRentals
 {
-    public class CarCRUD
+    public class CarCRUD : IDataProcessing
     {
-        const string PATH = @"Cars.json";
+        string PATH = ConfigurationManager.AppSettings["JsonFile"].ToString();
```

Comment:
this can be improved with https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-5.0